libmscore/element.cpp

```diff
@@ -375,12 +375,12 @@ QPointF Element::pagePos() const
 QPointF Element::canvasPos() const
       {
       QPointF p(pos());
-      if (parent() == 0)
+      if (parent() == nullptr)
             return p;
 
       if (_flags & ElementFlag::ON_STAFF) {
-            System* system = 0;
-            Measure* measure = 0;
+            System* system = nullptr;
+            Measure* measure = nullptr;
```

Comment:
I might be wrong, but isn't the default value of an uninitialised pointer `nullptr`?
So you could just write `System* system;`, for example?

Reply:
No.  Local variables need to be initialized.  Only globals / static variables get a predictable default value.

Reply:
Ah thanks, sorry about that.